Cargo.toml

```diff
@@ -22,18 +22,22 @@ rokid = ["rusb"]
 [dependencies]
 bytemuck = { version = "1.13.1", optional = true }
 byteorder = "1.4"
-nalgebra = { version = "0.32.3", default-features=false, features = ["std"]}
+nalgebra = { version = "0.32.3", default-features = false, features = ["std"] }
 rusb = { version = "0.9.2", optional = true }
 serialport = { version = "4.2", optional = true }
 tinyjson = { version = "2.5.1", optional = true }
+#yakf = { version = "0.1.12" }
 
 [target.'cfg(target_os = "android")'.dependencies]
-hidapi = { version = "2.4.1", default-features=false, features = [ "linux-static-libusb" ], optional = true }
+hidapi = { version = "2.4.1", default-features = false, features = ["linux-static-libusb"], optional = true }
 
 [target.'cfg(not(target_os = "android"))'.dependencies]
 hidapi = { version = "2.4.1", optional = true }
 
 
 [dev-dependencies]
 clap = { version = "4.3", features = ["derive"] }
-opencv = { version = "0.84.2", default-features = false, features = ["highgui", "imgproc", "calib3d"] }
+#opencv = { version = "0.84.2", default-features = false, features = ["highgui", "imgproc", "calib3d"] }
+
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
As I said below, C bindings should not be part of this crate.

It could be an optional feature somehow, or you should create a glue crate for your project.